ui-admin/src/integration/KitIntegrationDashboard.tsx

```diff
@@ -20,9 +23,26 @@ export default function KitIntegrationDashboard() {
     })
   }
 
+  const { isLoading } = useLoadingEffect(async () => {
+    const response = await Api.fetchInternalConfig()
+    setConfig(response)
+  })
+
   return <div>
-    <div>
-            Test kit status sync &nbsp;
+    <h2 className="h3">Kits</h2>
+    <div className="mt-4">
+      <h3 className="h5">Config</h3>
+      {!isLoading && <dl>
```

Comment:
Sorry, dumb question, but dsm config is constant across environments now and in the future?

Reply:
dsm config is configured via Spring Environment, which is picked up from application.yaml files, which in turn typically delegate to shell environment variables.  So the values are different in every environment.  

the names of the values are hardcoded.  I could see this instead implemented as a UI that just dumps the received config object on the screen, but I think it's helpful for the UI to indicate what config values are expected and be able to order them.

Reply:
Thx for that explanation.
I was think of sandbox, irb, live, so I probably used the wrong term?

Reply:
ah, yup, that was the right term -- I just misunderstood.  Yes, the idea is that sandbox/irb/live will interface with DSM in the same way, so that we can test "real" integrations with sandbox data (e.g. we can ship a kit to someone not enrolled in the study proper).  This might evolve as our integrations become more complex, and we want to fully isolate sandboxes from some external integrations

Reply:
I could see a useful feature where "useLiveDsm" is togglable on/off (maybe even defaulting off) for sandbox/irb environments so it's harder to accidentally ship a kit to a fake participant